src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,85 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    static final int DEFAULT_INITIAL_CAPACITY = 16;
+    static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private Node[] myTable = new Node[DEFAULT_INITIAL_CAPACITY];
+    private int size;
 
     @Override
     public void put(K key, V value) {
-
+        if (size == myTable.length * DEFAULT_LOAD_FACTOR) {
+            grow();
+        }
+        Node<K,V> newNode = myTable[getIndex(key)];
+        if (newNode == null) {
+            myTable[getIndex(key)] = new Node(key, value, null);
+            size++;
+        } else {
```

Comment:
Node<K, V> newNode = new Node<>(key, value, null);
        int index = getIndex(key);
  if (myTable[index]== null) {
            myTable[index] = newNode ;
            size++;
            return;
        }

Reply:
This is done in the same way, but without creating a new variable.

Reply:
you don't have to create an index. You do it twice:
Node<K, V> newNode = new Node<>(key, value, null);